CONTRIBUTING.md

```diff
@@ -8,10 +8,7 @@ financial contributions, issues, and pull requests!
 ### Financial contributions
 
 We welcome financial contributions in full transparency on our
-[open collective](https://opencollective.com/shields). Anyone can file an
-expense. If the expense makes sense for the development of the community, it
-will be "merged" into the ledger of our open collective by the core
-contributors and the person who filed the expense will be reimbursed.
+[open collective](https://opencollective.com/shields).
```

Comment:
I'm not aware of us having ever done this. Removing to reflect how we actually spend donations.